pandas/tests/extension/test_integer.py

```diff
@@ -57,6 +57,12 @@ def data_for_sorting(dtype):
     return integer_array([1, 2, 0], dtype=dtype)
 
 
+@pytest.fixture
+def data_multiple_nan(dtype):
```

Comment:
do we not already have a suitable fixture for this? or just change data_misssing_for_sorting to have another nan; adding this creates even more boilerplate and makes testing even more complex